optimum/exporters/neuron/model_configs.py

```diff
@@ -423,11 +428,144 @@ def inputs(self) -> List[str]:
         if self.task == "audio-xvector":
             common_outputs.append("embeddings")
         return common_outputs
 
 
+@register_in_tasks_manager(
+    "audio-spectrogram-transformer",
+    *[
+        "feature-extraction",
+        "audio-classification",
+    ],
+)
+class ASTNeuronConfig(AudioNeuronConfig):
+    NORMALIZED_CONFIG_CLASS = NormalizedConfig.with_args(
+        num_mel_bins="num_mel_bins", max_length="max_length", allow_new=True
+    )
+    DUMMY_INPUT_GENERATOR_CLASSES = (ASTDummyAudioInputGenerator,)
+
+    @property
+    def inputs(self) -> List[str]:
+        return ["input_values"]
+
+
+@register_in_tasks_manager(
+    "hubert",
+    *[
+        "feature-extraction",
+        "automatic-speech-recognition",
+        "audio-classification",
+    ],
+)
+class HubertNeuronConfig(Wav2Vec2NeuronConfig):
+    @property
+    def outputs(self) -> List[str]:
+        common_outputs = super().outputs
+        if self.task == "feature-extraction":
+            common_outputs = ["last_hidden_state"]
+        return common_outputs
+
+
+# TODO: compilation failed due to a bug in xla, https://github.com/huggingface/transformers/pull/31912 might be a workaround.
+# @register_in_tasks_manager(
+#     "sew",
+#     *[
+#         "feature-extraction",
+#         "automatic-speech-recognition",
+#         "audio-classification",
+#     ],
+# )
+# class SEWNeuronConfig(Wav2Vec2NeuronConfig):
+#     pass
+
+
+# TODO: compilation failed due to a bug in xla, https://github.com/huggingface/transformers/pull/31912 might be a workaround.
+# @register_in_tasks_manager(
+#     "sew-d",
+#     *[
+#         "feature-extraction",
+#         "automatic-speech-recognition",
+#         "audio-classification",
+#     ],
+# )
+# class SEWDNeuronConfig(Wav2Vec2NeuronConfig):
+#     pass
+
+
+@register_in_tasks_manager(
+    "unispeech",
+    *[
+        "feature-extraction",
+        "automatic-speech-recognition",
+        "audio-classification",
+    ],
+)
+class UniSpeechNeuronConfig(Wav2Vec2NeuronConfig):
+    pass
+
+
+@register_in_tasks_manager(
+    "unispeech-sat",
+    *[
+        "feature-extraction",
+        "automatic-speech-recognition",
+        "audio-classification",
+        "audio-frame-classification",
+        "audio-xvector",
+    ],
+)
+class UniSpeechSATNeuronConfig(Wav2Vec2NeuronConfig):
+    pass
+
+
+# TODO: compilation failed due to a bug in xla, https://github.com/huggingface/transformers/pull/31912 might be a workaround.
+# @register_in_tasks_manager(
+#     "wav2vec2-bert",
+#     *[
+#         "feature-extraction",
+#         "automatic-speech-recognition",
+#         "audio-classification",
+#         "audio-frame-classification",
+#         "audio-xvector",
+#     ],
+# )
+# class Wav2Vec2BertNeuronConfig(Wav2Vec2NeuronConfig):
+#     pass
+
+
+# TODO: compilation failed due to a bug in xla, https://github.com/huggingface/transformers/pull/31912 might be a workaround.
```

Comment:
I think the workaround has been rejected.

Reply:
yeah it is, we got to wait for the fix first on PyTorch XLA, then on Neuron SDK, it will be a long wait to go I guess... Considering if I should remove them...